.github/meta_issue_template/release-1.1.1-plugin-issues-min.md

```diff
@@ -0,0 +1,53 @@
+This is a component issue for release 1.1.1.
+Coming from [release issue 1.1.1](https://github.com/opensearch-project/opensearch-build/issues/870), release version 1.1.1. Please follow the following checklist.
+
+<details><summary>How to use this component issue</summary>
+<p>
+
+## This Component Issue
```

Comment:
Can we rename the these files to `components`?  Its the language used throughout the release process.  Plugin is a kind of component, but not all components are plugins like common-util

Reply:
Will do.